File: AndreasH-julia/Day5/part1.jl
using Printf
function extractBoardingId(boardingPass)
    return parse(Int,replace(replace(replace(replace(boardingPass,'F'=>"0"),'B'=>"1"),'L'=>"0"),'R'=>"1");base=2)
end
data = readlines(open("input.txt","r"))
highestID = 0
for line in data
    id = extractBoardingId(line)
    if id >  highestID
        global highestID = id
    end
end
@printf("Highest boarding pass ID: %d",highestID)

File: nattfalk-python/day6/part1.py
answered_questions = 0
with open('input.txt', 'r') as f:
    group_chars = ""
    for line in f:
        line = line.strip()
        if line:
            group_chars += line
        else:
            answered_questions += len(set(group_chars))
            group_chars = ""

if group_chars:
    answered_questions += len(set(group_chars))

print(f'Total {answered_questions}')